openapi.yaml

```diff
@@ -457,20 +457,53 @@ paths:
   '/applications/{app_id}/blockchain_version':
     summary: Get blockchain version of provided app_id
     get:
-      parameters: 
+      tags:
+        - Migration
+      parameters:
         - in: path
           name: app_id
           required: true
           schema:
-            type: string           
+            type: string
       responses:
         '200':
           description: OK
-          content: 
+          content:
             text/plain:
               schema:
                 type: string
-                example: pong
+                description: the version of blockchain the current app is working with
+                enum:
+                  - 2
+                  - 3
+  '/applications/{app_id}/migration-list':
```

Comment:
1. `migration_list`
2. I would do `/applications/{app_id}/migration/users` seems more RESTy and self-documenting

Reply:
Actually, @doodyparizada I thought about this. It might be even better to put any partner facing API in a completely different router and not with our v2 api as it is independent of it. Better yet, would be to move this to a new service. I would rename MP Public to MP external (vs MP internal) and create MP public for partner facing APIs (I have one I'm gonna add this week too). 

Of course, it should be a micro-service not related to MP at all, but we don't live in a perfect world just yet.